js/dom-utils/createCalc.js

```diff
@@ -0,0 +1,102 @@
+import { createEl } from "./createEl.js";
+
+export const createCalc = () => {
+    const content = document.getElementById('modalContent');
+
+    const calculatorDiv = createEl('div');
+    const displayDiv = createEl('div');
+    const calcButtonsDiv = createEl('div');
+    const cDiv = createEl('div');
+    const ceDiv = createEl('div');
+    const delDiv = createEl('div');
+    const slashDiv = createEl('div');
+    const sevenDiv = createEl('div');
+    const eightDiv = createEl('div');
+    const nineDiv = createEl('div');
+    const asteriscDiv = createEl('div');
+    const fourDiv = createEl('div');
+    const fiveDiv = createEl('div');
+    const sixDiv = createEl('div');
+    const minusDiv = createEl('div');
+    const oneDiv = createEl('div');
+    const twoDiv = createEl('div');
+    const threeDiv = createEl('div');
+    const plusDiv = createEl('div');
+    const zeroDiv = createEl('div');
+    const dotDiv = createEl('div');
+    const equalDiv = createEl('div');
+
+    displayDiv.setAttribute("id", "display");
+    equalDiv.setAttribute("id", "equal");
+
+    cDiv.innerText = 'C';
```

Comment:
store the text in an array, create a div for each element of the array, append text (string from the arary)
const butonCOntents = ["CE", "DEL"..]
buttonContents.map((btn) => {
   const div = createEl(div)
   text = document.createTExtNode(btn)
   div.appendChild(text)
  return div;
})

Reply:
select where you want to render the divs

divs.forEach(div => parent.appendChild(div))

Reply:
divs.forEach(div => div.classlist.add("button))